### ISSUE TITLE
Question on exercise output

### ISSUE THREAD

**Issue by amymhaddad** [amymhaddad/exercises_for_programmers#84]
I want to check if my output matches what this question is asking. 

Here are the instructions: Generate a single output statement with line breaks in the appropriate spots.
(I took that to mean to print all of the output together. Is this correct?)

import math

number_1 = int(input("Enter one number: "))
number_2 = int(input("Enter a second number: "))

sum = number_1 + number_2
difference = number_1 - number_2
product = number_1 * number_2
quotient = number_1 / number_2

print(f"""
The math operations are:
{number_1} + {number_2} = {sum}
{number_1} - {number_2} = {difference}
{number_1} * {number_2} = {product}
{number_1} / {number_2} = {math.ceil(quotient)}
""")

**Comment by paulghaddad** [comment]
Yes, this works. You could also print each one separately, with a separate `print` statement.
